README.rst

```diff
@@ -1,4 +1,123 @@
-Microsoft Azure SDK for Python
-==============================
+Microsoft Azure Storage SDK for Python
+======================================
 
-This is the Microsoft Azure Storage Runtime Client Library.
+This project provides a client library in Python that makes it easy to
+consume Microsoft Azure Storage services. For documentation please see
+the Microsoft Azure `Python Developer Center`_ and our `ReadTheDocs`_ page.
+
+    If you are looking for the Service Bus or Azure Management
+    libraries, please visit
+    https://github.com/Azure/azure-sdk-for-python.
+
+Features
+========
+
+-  Blob
+
+   -  Create/Read/Update/Delete containers
+   -  Create/Read/Update/Delete blobs
+   -  Advanced Blob Operations
+
+-  Queue
+
+   -  Create/Delete Queues
+   -  Insert/Peek Queue Messages
+   -  Advanced Queue Operations
+
+-  Table
+
+   -  Create/Read/Update/Delete tables
+   -  Create/Read/Update/Delete entities
+   -  Batch operations
+   -  Advanced Table Operations
+
+-  Files
+
+   -  Create/Update/Delete Directories
+   -  Create/Read/Update/Delete Files
+   -  Advanced File Operations
+
+Getting Started
+===============
+
+Download
+--------
+
+Option 1: Via PyPi
+~~~~~~~~~~~~~~~~~~
+
+To install via the Python Package Index (PyPI), type:
+::
+
+    pip.exe install azure-storage
+
+Option 2: Source Via Git
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To get the source code of the SDK via git just type:
+
+::
+
+    git clone git://github.com/Azure/azure-storage-python.git
+    cd ./azure-storage-python
+    python setup.py install
+
+Option 3: Source Zip
+~~~~~~~~~~~~~~~~~~~~
+
+To download a copy of the source code, click �Download ZIP� on the right
+side of the page or click `here`_. Unzip and navigate to the appropriate
```

Comment:
This readme will appear on the PyPI page as well.  While there's a .zip download over there too, the contents of the sources on github that you link to may not necessarily match the contents of the zip file they can click on the PyPI page.